### ISSUE TITLE
Add an explanation as to how to run the game to the DMG

### ISSUE THREAD

**Issue by cormacmccarthy** [deck-monsters/deck-monsters#265]
Explain the mechanics of how turns/rounds/etc works. This should describe both how Jane is intended to do it, and how someone using actual cards would play the game.

**Comment by tweettypography** [comment]
Makes sense. Also if we move command processing into this repo as mentioned in #259 it'll be trivial to add a more full-featured CLI test harness and a web interface.
